lldb/source/Breakpoint/BreakpointResolverName.cpp

```diff
@@ -24,10 +24,10 @@
 using namespace lldb;
 using namespace lldb_private;
 
-BreakpointResolverName::BreakpointResolverName(const BreakpointSP &bkpt,
```

Comment:
There are only reformattings in this file, unless I'm missing something.  Good to leave this sort of change out of patches that actually do something.

Reply:
Good catch, I didn't realize the formatter changes this file.